### PR TITLE
Added support for pkg-config

### PR DESCRIPTION
**Pull request by burlito**

### CODE REVIEW

**Comment by tjfontaine** [comment]
@saghul this seems reasonable to me, presuming the author signs the CLA

File: libuv.pc.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: @PACKAGE_NAME@
+Version: @PACKAGE_VERSION@
+Description:  multi-platform support library with a focus on asynchronous I/O.
```
Comment (saghul) [inline libuv.pc.in]: There seem to be 2 spaces after the colon.

**Comment by saghul** [comment]
LGTM (that minor nit aside), can you please sign the CLA? http://nodejs.org/cla.html

**Comment by burlito** [comment]
@saghul I already signed CLA :)

**Comment by saghul** [comment]
@burlito sweet! I reworded the commit message and landed it on https://github.com/joyent/libuv/commit/6c063317b74b6ac163d751cdeb3b3d3c43a6cffd Thanks a lot!

**Comment by mikemorris** [comment]
Any chance of this being backported to the [v0.10](https://github.com/joyent/libuv/tree/v0.10) branch?

**Comment by burlito** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

I have no idea, I'm not a maintainer but if you want this in v0.10 you
can try do this:

```
git checkout v0.10
wget https://github.com/joyent/libuv/pull/1026.patch
git am 1026.patch
```

This should do the trick.

On 08/20/2014 11:42 PM, Mike Morris wrote:

> Any chance of this being backported to the v0.10 
> https://github.com/joyent/libuv/tree/v0.10 branch?
> 
> — Reply to this email directly or view it on GitHub 
> https://github.com/joyent/libuv/pull/1026#issuecomment-52848830.
> 
> -----BEGIN PGP SIGNATURE-----
> Version: GnuPG v1

iQIcBAEBAgAGBQJT9RlOAAoJEMqQXu3yDIVasa0P/2ugVj9WLRdQXoe7/f0l1pcq
NjDT9CMZOGY+qfxIY6arCsfFrWInocHkEx2VJ48HI1rLKEK06isvxY5kK+MpQwSk
bFaxXLtOjidmouYnyQNGnpjTt3PxsqA3iHkWIvVPhuMEumQBnaJC6IwGqr+gbC+t
VP3BjPYTwJ+puyy2feiirwKWHQXPAfvzI1b/MJqRH31FBPjmS1i2s94lk0za2Gw8
RtHYNANYVv2KfSyQH/F4lwJlLdfzjzsswZFT3CVF1NcLP9tBilZ7mUEJPrjNK/tf
1XA0FO0Q9en7SW1MiLtuThnLZ/TV0FtEvxBFt8J1ClaAcNJ2tE+NBEzlY4SRwuw7
fHJ8l0h+bvGgIQtcRU0J2Ft2N76WGQ03BEJf+WGsrJNt+F5+L8/T71XFyoHtDHgA
HgToYGPvqNMplWl+YDsmr9GppLVRQFJRzVYbWqzHslDtlzKtza/9S//T7bOkMvKj
JtErdF0EvQao2ngMLH7Oxz4h8Lm9YUmuVupJTBfU6aLK/HcmLvUPauqYQzdO92b1
oWp+hp63Xd7/PfUP87tFQN034lctQ8kuV9LNu93in44h220mT2stsyG++MvI3+eZ
3rAyj1xLoLltzhY1kcVaCEMk/NGvofgl7vvuLctZw67VQsxVucjHLSm0GoNz6E5h
DSVJp2l+wWJsuORndbvP
=S6Nt
-----END PGP SIGNATURE-----

**Comment by saghul** [comment]
v0.10 only gets bug fixes at the moment. While its debatable if this is a new feature, I'm -1 on backporting it.

**Comment by txdv** [comment]
It is not really a code feature. More of a packaging feature.

**Comment by saghul** [comment]
0.10 doesn't use autoconf, so backporting this is not just applying the patch.

**Comment by txdv** [comment]
Too much trouble for something ancient like 0.10

**Pull request closed without merging**